Library/Formula/emacs.rb

```diff
@@ -39,6 +40,7 @@ class Emacs < Formula
   depends_on "imagemagick" => :optional
   depends_on "mailutils" => :optional
   depends_on "glib" => :optional
+  depends_on "libxml2" => :optional
```

Comment:
It can and works fine, just wondering does that mean I remove this line completely what if the user doesn't have `libxml2` installed (I know that's unlikely, but I just wanted to make sure the build doesn't break for such a reason - or is this just expected behaviour in that case?)